packages/cadl-ranch-specs/http/dictionary/main.cadl

```diff
@@ -0,0 +1,99 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-ranch-expect";
+
+@doc("Illustrates various of dictionary.")
+@scenarioService("/dictionary")
+namespace Dictionary;
+using Cadl.Http;
+
+// TEMPLATES
+@doc("Template type for testing dictionary with specific value type. Pass in the type of the value you are looking for")
+model ModelValueTemplate<TProperty> {
+  @doc("Property")
+  property: Record<TProperty>;
+}
+
+@doc("Template to have models operations")
+interface ModelOperations<TModel, TDoc extends string> {
+  @scenario
+  @scenarioDoc(
+    """
+  Expected response body:
+  ```json
+  {"property": {doc}}
+  ```
+  """,
+    {
+      doc: TDoc,
+    }
+  )
+  @get
+  get(): TModel;
+
+  @scenario
+  @scenarioDoc(
+    """
+  Expected input body:
+  ```json
+  {"property": {doc}}
+  ```
+  """,
+    {
+      doc: TDoc,
+    }
+  )
+  @put
+  put(@body body: TModel): OkResponse;
+}
+
+// Test a model with an int dictionary property
+@doc("Model with dictionary int properties")
```

Comment:
@timotheeguerin correct me if I'm wrong, but I think for the dictionary tests, we really want to be focusing on testing dictionaries themselves, not dictionaries as properties. We can add dictionary property tests to the `property-types` test, but for the dictionary test we don't want them all to be tied to models